tools/github-actions/cascading/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Cascading'
+description: 'Cascade the commit(s) after a merge on the next branches based on semver and default branch'
+inputs:
+  noFf:
+    description: 'Force a merge commit in the cascading'
+    required: false
+    default: false
+  cascadingPattern:
+    description: 'The cascading pattern that needs to be applied'
+    required: false
+    default: 'release/*'
+  ignoredPattern:
+    description: 'Ignore the branches that match this pattern for the cascading ex: -next$'
+    required: false
+  defaultBranch:
+    description: 'The default branch if you have one (ex: master, development), if no candidate found with the given pattern this branch will be the last one where the code will be cascaded'
+    required: false
+    default: 'main'
+  assignCommitter:
+    description: 'Assigns the first committer to the cascading failure PR'
+    required: false
+    default: false
+  token:
+    description: 'Github token to use to access to the APIs'
```

Comment:
I think you need to specify the the permission(s) mandatory to this token (here or in the readme)